From f6545b0f72fe27208e8cc719db3573760cc40c11 Mon Sep 17 00:00:00 2001
From: Tomas Zak <tomas.zak@turris.com>
Date: Wed, 11 Dec 2024 18:54:09 +0100
Subject: [PATCH] patches/packages/python-cryptography: bump libc for
 cryptography lib

Due to multiple definitions of variable MAP_HUGETLB in libc for powerpc, bump
from version 0.2.144 to 0.2.148 where this issue is fixed
---
 ...-bump-libc-popwerpc-multiple-definition.patch | 16 ++++++++++++++++
 1 file changed, 16 insertions(+)
 create mode 100644 lang/python/python-cryptography/patches/002-bump-libc-popwerpc-multiple-definition.patch

diff --git a/lang/python/python-cryptography/patches/002-bump-libc-popwerpc-multiple-definition.patch b/lang/python/python-cryptography/patches/002-bump-libc-popwerpc-multiple-definition.patch
new file mode 100644
index 000000000..5aed8fe5d
--- /dev/null
+++ b/lang/python/python-cryptography/patches/002-bump-libc-popwerpc-multiple-definition.patch
@@ -0,0 +1,16 @@
+Index: cryptography-41.0.7/src/rust/Cargo.lock
+===================================================================
+--- cryptography-41.0.7.orig/src/rust/Cargo.lock
++++ cryptography-41.0.7/src/rust/Cargo.lock
+@@ -137,9 +137,9 @@ checksum = "bfa799dd5ed20a7e349f3b4639aa
+ 
+ [[package]]
+ name = "libc"
+-version = "0.2.144"
++version = "0.2.148"
+ source = "registry+https://github.com/rust-lang/crates.io-index"
+-checksum = "2b00cc1c228a6782d0f076e7b232802e0c5689d41bb5df366f2a6b6621cfdfe1"
++checksum = "9cdc71e17332e86d2e1d38c1f99edcb6288ee11b815fb1a4b049eaa2114d369b"
+ 
+ [[package]]
+ name = "lock_api"
-- 
2.47.1

